core/store/migrate/migrations/0081_unconsumed_log_broadcasts.sql

```diff
@@ -0,0 +1,17 @@
+-- +goose Up
+ALTER TABLE log_broadcasts ADD COLUMN updated_at timestamp with time zone NOT NULL;
+DROP INDEX IF EXISTS log_consumptions_unique_v2_idx;
+CREATE UNIQUE INDEX log_broadcasts_unique_idx ON log_broadcasts(job_id, block_hash, log_index, evm_chain_id);
```

Comment:
You can make this index smaller by adding `where job_id is not null` condition.

Assuming we don't care about using it to speed up lookups of explicitly null job_ids

Reply:
Can't simply drop it since it's used for an `INSERT ON CONFLICT`, but could instead tighten up the column constraints to `job_id IS NOT NULL` since that's not being used anyways.